src/mono/netcore/sample/iOS/Plist.in

```diff
@@ -25,10 +25,6 @@
 		<key>UIApplicationSupportsMultipleScenes</key>
 		<false/>
 	</dict>
-	<key>UIRequiredDeviceCapabilities</key>
-	<array>
-		<string>arm64</string>
-	</array>
```

Comment:
did you check whether it still works on device? I remember there was some issue when this key was removed

Reply:
@akoeplinger yeah I checked, still works for both device and simulator. I think it was `armv7` by default and I changed it to `arm64`. I've just created a simple project in Xcode and this property don't even exist in the default plist.